gapipy/request.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, client, resource, params=None, parent=None):
     def _request(self, uri, method, data=None, params=None, additional_headers=None):
         """Make an HTTP request to a target API method with proper headers."""
 
-        assert method in ['GET', 'POST', 'PUT', 'PATCH'], "Only 'GET', 'POST', 'PUT', and 'PATCH' are allowed."
+        assert method in ['GET', 'POST', 'PUT', 'PATCH', 'OPTIONS'], "Only 'GET', 'POST', 'PUT', 'PATCH' and 'OPTIONS' are allowed."
```

Comment:
This is indeed a bit long. Really not an issue, but we could probably write it as such:

```
# place at top with other constants
ALLOWED_METHODS = ['GET', 'POST', 'PUT', 'PATCH', 'OPTIONS']

assert method in ALLOWED_METHODS, "Only {} are allowed.".format(', '.join(ALLOWED_METHODS))
```